Changing to `pep8 code` yields some more issues that don't seem to have been addressed in #79:

```
% make check                                                                                                                  [~/Code/Web/first-analysis-steps]
python tools/check.py .
pep8 code
code/dst-explore.py:20:1: W293 blank line contains whitespace
code/dst-explore.py:32:1: W293 blank line contains whitespace
code/dst-explore.py:35:1: W293 blank line contains whitespace
code/dst-explore.py:41:1: E302 expected 2 blank lines, found 1
code/dst-explore.py:48:12: E225 missing whitespace around operator
code/dst-explore.py:49:53: E228 missing whitespace around modulo operator
code/dst-explore.py:57:15: E225 missing whitespace around operator
code/dst-explore.py:59:1: E402 module level import not at top of file
code/dst-explore.py:72:26: W291 trailing whitespace
code/dst-explore.py:75:1: E265 block comment should start with '# '
code/dst-explore.py:75:25: W291 trailing whitespace
code/dst-explore.py:76:1: E265 block comment should start with '# '
code/dst-explore.py:81:1: W293 blank line contains whitespace
code/dst-explore.py:87:80: E501 line too long (92 > 79 characters)
code/dst-explore.py:90:80: E501 line too long (86 > 79 characters)
code/dst-explore.py:92:80: E501 line too long (83 > 79 characters)
code/dst-explore.py:95:1: E265 block comment should start with '# '
code/dst-explore.py:95:80: E501 line too long (101 > 79 characters)
code/dst-explore.py:97:80: E501 line too long (86 > 79 characters)
code/dst-explore.py:120:1: E265 block comment should start with '# '
code/dst-explore.py:122:1: E265 block comment should start with '# '
code/dst-explore.py:122:80: E501 line too long (82 > 79 characters)
code/dst-explore.py:123:1: E265 block comment should start with '# '
code/dst-explore.py:124:1: E265 block comment should start with '# '
code/dst-explore.py:126:1: E265 block comment should start with '# '
code/dst-explore.py:130:1: E265 block comment should start with '# '
code/dst-explore.py:131:1: E265 block comment should start with '# '
code/dst-explore.py:133:1: E265 block comment should start with '# '
code/12-add-tupletools/ntuple_options.py:17:25: E203 whitespace before ':'
code/12-add-tupletools/ntuple_options.py:18:25: E203 whitespace before ':'
code/12-add-tupletools/ntuple_options_loki.py:17:25: E203 whitespace before ':'
code/12-add-tupletools/ntuple_options_loki.py:18:25: E203 whitespace before ':'
code/12-add-tupletools/ntuple_options_loki.py:42:80: E501 line too long (93 > 79 characters)
code/11-davinci-grid/first-job.py:4:80: E501 line too long (187 > 79 characters)
code/14-rerun-stripping/options.py:33:1: E402 module level import not at top of file
code/14-rerun-stripping/options.py:41:1: E402 module level import not at top of file
code/14-rerun-stripping/options.py:42:1: E402 module level import not at top of file
code/14-rerun-stripping/options.py:43:1: E402 module level import not at top of file
code/18-switch-mass-hypo/ntuple_switchHypo.py:11:1: E402 module level import not at top of file
code/18-switch-mass-hypo/ntuple_switchHypo.py:12:1: E402 module level import not at top of file
code/18-switch-mass-hypo/ntuple_switchHypo.py:13:1: E402 module level import not at top of file
code/18-switch-mass-hypo/ntuple_switchHypo.py:15:48: W291 trailing whitespace
code/18-switch-mass-hypo/ntuple_switchHypo.py:17:80: E501 line too long (80 > 79 characters)
code/18-switch-mass-hypo/ntuple_switchHypo.py:17:81: W291 trailing whitespace
code/18-switch-mass-hypo/ntuple_switchHypo.py:18:1: E402 module level import not at top of file
code/18-switch-mass-hypo/ntuple_switchHypo.py:21:13: E251 unexpected spaces around keyword / parameter equals
code/18-switch-mass-hypo/ntuple_switchHypo.py:21:15: E251 unexpected spaces around keyword / parameter equals
code/18-switch-mass-hypo/ntuple_switchHypo.py:22:22: E251 unexpected spaces around keyword / parameter equals
code/18-switch-mass-hypo/ntuple_switchHypo.py:22:24: E251 unexpected spaces around keyword / parameter equals
code/18-switch-mass-hypo/ntuple_switchHypo.py:22:26: E261 at least two spaces before inline comment
code/18-switch-mass-hypo/ntuple_switchHypo.py:23:9: E122 continuation line missing indentation or outdented
code/18-switch-mass-hypo/ntuple_switchHypo.py:23:41: E203 whitespace before ':'
code/18-switch-mass-hypo/ntuple_switchHypo.py:23:50: W291 trailing whitespace
code/18-switch-mass-hypo/ntuple_switchHypo.py:24:9: E122 continuation line missing indentation or outdented
code/18-switch-mass-hypo/ntuple_switchHypo.py:24:41: E203 whitespace before ':'
code/18-switch-mass-hypo/ntuple_switchHypo.py:24:50: W291 trailing whitespace
code/18-switch-mass-hypo/ntuple_switchHypo.py:29:80: E501 line too long (92 > 79 characters)
code/12-split-jobs/first-job.py:4:80: E501 line too long (187 > 79 characters)
Makefile:36: recipe for target 'check' failed
make: *** [check] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -34,6 +34,7 @@ all : commands
 ## check    : Validate all lesson content against the template.
 check: $(ALL_MD)
 	python tools/check.py .
+	pep8 code/*.py
 
 ## clean    : Clean up temporary and intermediate files.
 clean :

--- a/Makefile
+++ b/Makefile
@@ -34,7 +34,7 @@ all : commands
 ## check    : Validate all lesson content against the template.
 check: $(ALL_MD)
 	python tools/check.py .
-	pep8 code/*.py
+	pep8 code
 
 ## clean    : Clean up temporary and intermediate files.
 clean :

--- a/code/22-decay-tree-fitter/ntuple_DTF1.py
+++ b/code/22-decay-tree-fitter/ntuple_DTF1.py
@@ -1,5 +1,9 @@
 from GaudiConf import IOHelper
-from Configurables import DaVinci, DecayTreeTuple
+from Configurables import (
+    DaVinci,
+    DecayTreeTuple,
+    TupleToolDecayTreeFitter
+)
 from DecayTreeTuple.Configuration import *
 
 # Stream and stripping line we want to use
@@ -12,7 +16,6 @@ dtt.Inputs = ['/Event/{0}/Phys/{1}/Particles'.format(stream, line)]
 dtt.Decay = '[D*(2010)+ -> ^(D0 -> ^K- ^pi+) ^pi+]CC'
 
 # add a kinematic fitter
-from Configurables import TupleToolDecayTreeFitter
 dtt.addBranches({
     'Dstar': '[D*(2010)+ -> (D0 -> K- pi+) pi+]CC',
 })

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,4 @@
 CommonMark
 pandocfilters
 PyYAML
+pep8

